exercises/isbn-verifier/description.md

```diff
@@ -1,4 +1,5 @@
-Check if a given ISBN-10 is valid.
+The [ISBN-10 verification process](http://www.hahnlibrary.net/libraries/isbncalc.html) is mostly used to validate book identification
```

Comment:
I'd remove this link, and replace it with the wikipedia one. This link doesn't give much useful information about the number and the formula given there conflicts with the one specified here. (1-9) are reversed on that page.

Reply:
Ah, I didn't notice that! On it.